vnda/loaders/productList.ts

```diff
@@ -42,6 +45,10 @@ const productListLoader = async (
       sort: props?.sort,
       per_page: props?.count,
       "tags[]": props?.tags,
+      "type_tags[]": props?.type_tags?.map((tag) => ({
+        key: tag.key,
+        value: tag.value,
+      })),
```

Comment:
Acho que nao precisa do map aqui pq voce ta transformando o `type_tags` na mesma coisa. 
```suggestion
      "type_tags[]": props?.type_tags ?? [],,
```